test/multiverse/suites/rack/Envfile

```diff
@@ -14,10 +14,10 @@ PUMA_VERSIONS = [
 ]
 
 def gem_list(puma_version = nil)
-  rack_version = RUBY_VERSION >= '2.3.0' ? '~> 2.2.3' : '~> 2.1.4'
+  rack_version = ' ~> 2.1.4' unless RUBY_VERSION >= '2.3.0'
   <<-RB
     gem 'puma'#{puma_version}
-    gem 'rack', '#{rack_version}'
```

Comment:
The new implementation is missing an `, '` to correctly separate the version number from the gem name. Without this, a bundler syntax error is thrown (which is where the CI Rack failures are coming from).  Since we're down to just one version, perhaps adding the `, '` to the `rack_version` variable assignment would look cleaner than keeping it on line 20?

Reply:
Thank you! Addressed with [cfe4922](https://github.com/newrelic/newrelic-ruby-agent/pull/1600/commits/cfe4922ffcad9b540cd8bb14e827c14e32bc811f)